platform/accessibility/vsp-accessibility-rules-of-engagement.md

```diff
@@ -0,0 +1,93 @@
+
+   * Relevant design system components and patterns
+
+   * Feedback on recruiting diverse research participants
+
+   * Additional testing scenarios
+
+4. **Staging Review** — *Integrated with [VSP design QA review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/design/working-with-platform-design-team.md#design-reviews-1)*
+
+   * The accessibility specialist will review the following items:
+     * A completed test plan. This test plan could be stored in [TestRail](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) **or** [Github](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md). **Incomplete test plans can delay the staging accessibility review.**
+     * Automated end-to-end (e2e) tests for client-side applications.
+       * The specialist will check that tests have been written, and include sufficient axe checks for the relevant user interactions.
+       * Axe checks should be run for all views, and for hidden content like modal windows and accordions.
+   * The accessibility specialist test the [happy path](https://en.wikipedia.org/wiki/Happy_path) using the following methodology:
+     * Zooming layouts to 400% at 1280px width
+     * Keyboard navigation
+     * Three preferred screen reader and browser pairings. Pairings will be determined by Google Analytics data and WebAIM’s [most recent screen reader survey](https://webaim.org/projects/screenreadersurvey8/#primary). This will most likely include:
+       * JAWS + IE11 or Chrome on Windows 7/10
+       * NVDA + Firefox on Windows 7/10
+       * VoiceOver on MacOS latest _(Mojave or Catalina as of February 2020)_
+     * Any issues will be collected in a Zenhub epic
+     * Issues will be triaged for severity, and pre-launch or post-launch fixes
+
+5. **Full Accessibility Audit and 508 Office Review**
+
+   * **508 Post-launch Audit**<br/>
+     Applications still need a full accessibility audit. This audit was previously done during the pre-launch phase, but will now be done immediately post-launch to emphasize speed and agility. Any findings during the post-launch audit will be forwarded to the VA 508 office as known items. This should prevent duplicate logging when the VA 508 office conducts their own test.
+
+     Post-launch audits will be beneficial for finding usability issues that would not otherwise appear on an automated report. These findings will drive future innovation and research sessions. Findings will also drive improvements to the design system and overall accessibility guidance.
+
+     The VSP post-launch audit will focus on manual testing with mobile and assistive technology:
+
```

Comment:
I listed out the step-by-step of the pre-launch 508 review, perhaps it would be helpful to list these steps here?

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/accessibility-dev-review-step-by-step.md